File: docker/src/assistance/model/entities/Dispositivo.py
from sqlalchemy import Column, String, Integer, DateTime, ForeignKey, Boolean, Date, DateTime
from sqlalchemy.orm import relationship
from model_utils import Base

class Dispositivo(Base):

    __tablename__ = 'dispositivo'
    __table_args__ = {'schema':'assistance'}

    nombre = Column(String)
    ip = Column(String)
    mascara = Column(String)
    router = Column(String)
    mac = Column(String)
    modelo = Column(String)
    serial = Column(String)
    fecha_del_producto = Column(DateTime)
    url = Column(String)
    algoritmo = Column(String)
    habilitado = Column(Boolean)
    descripcion = Column(String)
    eliminado = Column(DateTime)

File: docker/src/assistance/web/src/app/entities/dispositivo.ts



export class Dispositivo {

  id: string = null;
  creado: Date = null;
  actualizado: Date = null;

  nombre: string = null;
  ip: string;
  mascara: string;
  router: string;
  mac: string;
  modelo: string;
  serial: string;
  fecha_del_producto: Date;
  url: string;
  algoritmo: string;
  habilitado: boolean;

  constructor(o:Object) {
    if (o) {
      Object.assign(this, o);

    this.fecha_del_producto = (this.fecha_del_producto == null ? null : new Date(this.fecha_del_producto));}
    this.creado = (this.creado == null ? null : new Date(this.creado));
    this.actualizado = (this.actualizado == null ? null : new Date(this.actualizado));
  }


}
